lib/recovery/recover_lost_jobs.rb

```diff
@@ -0,0 +1,8 @@
+module Recovery
```

Comment:
I think we should switch this around a bit: a single worker that calls a bunch of classes. We'll want to extend this to support other kinds of recovery.